backvendor/gosource.go

```diff
@@ -429,26 +437,46 @@ func (src GoSource) Vendor() string {
 	return filepath.Join(src.Path, "vendor")
 }
 
-// Project returns information about the project given its import
+// Project returns information about the project's repository, as well
+// as the project's path within the repository, given its import
 // path. If importPath is "" it is deduced from import comments, if
 // available.
-func (src GoSource) Project(importPath string) (*RepoRoot, error) {
+func (src GoSource) Project(importPath string) (*RepoPath, error) {
 	if importPath == "" {
 		importPath = src.Package
 		if importPath == "" {
 			return nil, ErrorNeedImportPath
 		}
 	}
 
-	repoRoot, err := vcs.RepoRootForImportPath(importPath, false)
-	return &RepoRoot{RepoRoot: *repoRoot}, err
+	repoRoot, err := vcsRepoRootForImportPath(importPath, false)
+	if err != nil {
+		return nil, err
+	}
+
+	// Work out root path
+	r := repoRoot.Root
+	var rootPath string
+	switch {
+	case importPath == repoRoot.Root:
```

Comment:
`importPath == repoRoot.Root`  => `importPath == r` for consistency.

Reply:
Thanks, fixed.